docs/source/user_guide/faq.rst

```diff
@@ -0,0 +1,118 @@
+fastplotlib FAQ
+===============
+
+What is `fastplotlib`?
+----------------------
+
+    `fastplotlib` is a scientific plotting library built on top of the `pygfx <https://github.com/pygfx/pygfx>`_ rendering engine
+    that leverages new graphics APIs and modern GPU hardware to create fast and interactive visualizations.
+
+
+What can I do with `fastplotlib`?
+---------------------------------
+
+    `fastplotlib` allows for:
+        - GPU accelerated visualization
+        - interactive visualization via an intuitive and expressive API
+        - rapid prototyping and algorithm design
+        - easy exploration and fast rendering of large-scale data
+        - design, develop, evaluate and ship machine learning models
+
```

Comment:
add "create visualizations for real-time acquisition systems for scientific instruments, such as cameras and other devices"